sklearn/utils/random.py

```diff
@@ -23,8 +23,9 @@ def _random_choice_csc(n_samples, classes, class_probability=None,
     classes : list of size n_outputs of arrays of size (n_classes,)
         List of classes for each column.
 
-    class_probability : list of size n_outputs of arrays of size (n_classes,)
-        Optional (default=None). Class distribution of each column. If None the
+    class_probability : list of size n_outputs of arrays of size (n_classes,),
+        default=None. 
+        Class distribution of each column. If None the
         uniform distribution is assumed.
```

Comment:
We can move "uniform" up and take advantage of the 80 characters we have per line.

Reply:
Absolutely. Thanks @thomasjpfan .